src/shared/exchange/xgrid.F90

```diff
@@ -477,9 +477,15 @@ module xgrid_mod
  integer :: id_get_2_from_xgrid = 0
  integer :: id_put_2_to_xgrid = 0
  integer :: id_setup_xmap = 0
- integer :: id_load_xgrid1, id_load_xgrid2, id_load_xgrid3
- integer :: id_load_xgrid4, id_load_xgrid5
- integer :: id_load_xgrid, id_set_comm, id_regen, id_conservation_check
+ integer :: id_load_xgrid = 0
```

Comment:
Another option for these is to only call the timer functions if `xgrid_clocks_on` is set, e.g
```fortran
if(xgrid_clocks_on) call mpp_clock_begin(id_load_xgrid1)
```

Reply:
I see `app_clock_begin` returns if the `id == 0`, so yeah, same same.